tests/api/test_verification_case.py

```diff
@@ -0,0 +1,546 @@
+            ).validate_verification_case_structure(case_missing_subject)
+            == False
+        )
+
+        case_missing_verification_class = {
+            "no": 2,
+            "run_simulation": True,
+            "simulation_IO": {
+                "idf": "../test_cases/doe_prototype_cases/ASHRAE901_Hospital_STD2004_Atlanta_Case2.idf",
+                "idd": "../resources/Energy+V9_0_1.idd",
+                "weather": "../weather/USA_GA_Atlanta-Hartsfield.Jackson.Intl.AP.722190_TMY3.epw",
+                "output": "eplusout.csv",
+                "ep_path": "C:\\EnergyPlusV9-0-1\\energyplus.exe",
+            },
+            "expected_result": "fail",
+            "datapoints_source": {
+                "idf_output_variables": {
+                    "T_sa_set": {
+                        "subject": "VAV_1 Supply Equipment Outlet Node",
+                        "variable": "System Node Setpoint Temperature",
+                        "frequency": "detailed",
+                    }
+                },
+                "parameters": {"T_z_coo": 24.0},
+            },
+            "verification_class": [
+                "SupplyAirTempReset"
+            ],  # intentionally put list instead of str
+        }
+
+        assert (
+            VerificationCase(
+                case=None, file_path=None
+            ).validate_verification_case_structure(case_missing_verification_class)
+            == False
+        )
+
+    def test_save_verification_cases_to_json(self):
+        example_cases = [
+            {
+                "no": 1,
+                "run_simulation": True,
+                "simulation_IO": {
+                    "idf": "../test_cases/doe_prototype_cases/ASHRAE901_Hospital_STD2004_Atlanta_Case2.idf",
+                    "idd": "../resources/Energy+V9_0_1.idd",
+                    "weather": "../weather/USA_GA_Atlanta-Hartsfield.Jackson.Intl.AP.722190_TMY3.epw",
+                    "output": "eplusout.csv",
+                    "ep_path": "C:\\EnergyPlusV9-0-1\\energyplus.exe",
+                },
+                "expected_result": "fail",
+                "datapoints_source": {
+                    "idf_output_variables": {
+                        "T_sa_set": {
+                            "subject": "VAV_1 Supply Equipment Outlet Node",
+                            "variable": "System Node Setpoint Temperature",
+                            "frequency": "detailed",
+                        }
+                    },
+                    "parameters": {"T_z_coo": 24.0},
+                },
+                "verification_class": "SupplyAirTempReset",
+            }
+        ]
+        json_path = "./tests/api/result/wrong_json_path.json"
+
+        # when wrong json_path type is provided
+        with self.assertLogs() as logobs:
+            wrong_json_path = ["./wrong_json_path.json"]
+            VerificationCase(case=None, file_path=None).save_verification_cases_to_json(
```

Comment:
to make sure this method is a static method, test without instantiate an object. This also applies to tests of other static methods.

Reply:
Is there any way to use a staticmethod without object instantiation? Could you show me one example?

Reply:
yes, just ClassName.method_name(args)

Reply:
Good point! Fixed.